project/init.php

```diff
@@ -140,7 +140,7 @@ function get_mttinfo($v)
 			if($_mttinfo['mtt_url'] == '') $_mttinfo['mtt_url'] = url_dir(isset($_SERVER['REQUEST_URI']) ? $_SERVER['REQUEST_URI'] : $_SERVER['SCRIPT_NAME']);
 			return $_mttinfo['mtt_url'];
 		case 'title':
-			$_mttinfo['title'] = (Config::get('title') != '') ? htmlarray(Config::get('title')) : __('My Tiny Todolist');
```

Comment:
Looks Great!  After you test on your local machine , I will merge your pull request